doc/ug/getting_started/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Getting Started"
+---
+
+Welcome!
+
+This guide will help you get OpenTitan up and running by instructing you how to:
+
+1. setup an adequate build/testing environment on your machine,
+2. clone the OpenTitan Git repository, and
+3. build OpenTitan software/hardware for the target of your choosing.
+
+![Getting Started Workflow](getting_started_workflow.svg)
+
+As shown in the diagram above, we currently support multiple build targets and workflows, including: Verilator, FPGA, and DV.
+However, if you are new to the project, we recommend trying the Verilator workflow, as this uses only free EDA tools, and does not require any additional hardware.
+
+## Environment Setup
+
+There are two choices for setting up an OpenTitan environment:
+1. \[Supported\] [manual installation]({{< relref "install_instructions" >}}), or
+2. \[Experimental\] [provided Docker container]({{< relref "/util/container/README.md" >}}).
+
+While the first option is the only one currently supported, the second option is the quickest to get running.
+If you plan on being a long term user of the project, and/or contributer, we recommend the first (manual installation) option.
+Alternatively, if your goal is to get a Verilator simulation up and running as quickly as possible, we recommend the second (container) option.
+
+## Cloning the OpenTitan Repository
+
+The [OpenTitan Repository](https://github.com/lowRISC/opentitan) must be checked out locally.
+If you wish to contribute to OpenTitan you will need to make a fork on GitHub, otherwise you can just locally clone the main repository.
+There are a set of [notes for using GitHub]({{< relref "github_notes.md" >}}) which explains how to work with your own fork.
+
+***Note: throughout the documentation `$REPO_TOP` refers to the path where the OpenTitan repository is checked out.***
+
+## Building OpenTitan Software/Hardware
+
+Once you have setup a suitable environment and cloned the OpenTitan source code, you need to choose a top-level OpenTitan design to build.
+The code base contains multiple top-level designs, listed [here]({{< relref "/doc/ug/system_list.md" >}}), which can be synthesized or compiled for various targets.
+A target can be a simulation tool, an FPGA board, or an ASIC technology.
+We currently support two out of the three aformentioned target types (i.e., simulation tools and FPGA), as described in this table:
```

Comment:
I know the wording we have in many places is blurry, but let's at least clarify things slightly here what "Verilator" vs "DV" means:

* What is called here "Verilator" is a cycle-accurate chip simulation primarily aimed at software development. (Yes, it's produced by Verilator, but that's mostly an implementation detail.)
* What is called "DV" is the hardware verification environment, primarily a constraint-random verification environment for individual hardware blocks. It also provide a full-chip simulation, but we use that mostly for testing the hardware in combination with some software, not developing software itself. The "DV" simulation can be powered by different simulators, VCS and Xcelium being the two primary ones.

Reply:
@imphil I updated the table to indicate the primary purpose of each target. Let me know if you think this is OK? or if I should remove it?

Reply:
Thanks for the update, looks good!